dataprofiler/tests/profilers/test_float_options.py

```diff
@@ -0,0 +1,28 @@
+from dataprofiler.profilers.profiler_options import FloatOptions
+from dataprofiler.tests.profilers.test_numerical_options import TestNumericalOptions
+
+class TestFloatOptions(TestNumericalOptions):
+    
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.keys += ["precision"]
+        cls.option_class = FloatOptions    
```

Comment:
In a case like this, I would expect it to be the top level, so having set outside of the setup class seems reasonable to me